pkg/api/server.go

```diff
@@ -38,37 +39,66 @@ var uiBox *packr.Box
 //var legacyUiBox *packr.Box
 var setupUIBox *packr.Box
 
+func allowUnauthenticated(r *http.Request) bool {
+	return r.URL.Path == "/login" || r.URL.Path == "/css" || strings.HasPrefix(r.URL.Path, "/setup")
```

Comment:
Adding /setup here poses a security risk
Anyone can access and mess with the setup even if a user/password is set

Reply:
Moved login files to a separate packr box and amended the line. No longer possible to access `/setup` without logging in.